core-runner/src/runner.rs

```diff
@@ -339,11 +297,11 @@ impl<SC: StorageCarrier, E: Environment<Ext>> Runner<SC, E> {
                 gas_limit,
                 max_gas_limit
             );
-            self.message_queue.queue(next_message);
+            self.message_queue.push(message);
             return RunNextResult::empty();
         }
 
-        let instrumeted_code = match gas::instrument(program.code()) {
+        let instrumented_code = match gas::instrument(program.code()) {
```

Comment:
Might be write simpler with `.map_err(|err| { .. })?` syntax

Reply:
Agree but I've just fixed a typo here :-) Let's do it later.